feconf.py

```diff
@@ -426,6 +426,8 @@ def get_empty_ratings():
 OPPIA_PROJECT_ID = 'dev-project-id'
 GOOGLE_APP_ENGINE_REGION = 'us-central1'
 
+FIREBASE_AUTH_EMULATOR_HOST = 'localhost:9099'
```

Comment:
Normally we separate host and port into two constants (see e.g.redis). OK to do it like this if you prefer but maybe call it ..._HOST_AND_PORT for consistency?

Reply:
Hmm I'm concerned giving it a different name than we assign to os.environ will make people suspect there's a bug.

I added a comment + reference to help highlight that this naming is intentional. WDYT?

Reply:
OK, seems fine to me, thanks for explaining. Made an extra suggestion to clarify the comment.

Reply:
Done.